service-runner/src/main/resources/profiles/host-docker-internal/config.env

```diff
@@ -1,3 +1,4 @@
 sep10.home_domain=host.docker.internal:8080
 sep1.toml.value=/config/stellar.host.docker.internal.toml
-sep24.more_info_url.base_url=http://host.docker.internal:8091/sep24/transaction/more_info
\ No newline at end of file
+sep24.more_info_url.base_url=http://host.docker.internal:8091/sep24/transaction/more_info
+wallet.server.url=http://host.docker.internal:8092
```

Comment:
Do we still need this? I believe you added this initially to fix the wallet server.

Reply:
I don't think so